polaris-icons/icons/PersonalizedTextMajor.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none"><g fill="#616A75" clip-path="url(#a)"><path d="M2.707 9.707A.996.996 0 0 0 3 9V5a1 1 0 0 1 1-1 1 1 0 0 0 0-2C2.346 2 1 3.346 1 5v3.586l-.707.707a.999.999 0 0 0 0 1.414l.707.707V15c0 1.654 1.346 3 3 3a1 1 0 0 0 0-2 1 1 0 0 1-1-1v-4a.997.997 0 0 0-.293-.707L2.414 10l.293-.293ZM19.924 9.617a1 1 0 0 0-.217-.324L19 8.586V5c0-1.654-1.346-3-3-3a1 1 0 1 0 0 2 1 1 0 0 1 1 1v4a.996.996 0 0 0 .293.707l.293.293-.293.293A.996.996 0 0 0 17 11v4a1 1 0 0 1-1 1 1 1 0 0 0 0 2c1.654 0 3-1.346 3-3v-3.586l.707-.707a1 1 0 0 0 .217-1.09ZM10 9a2.5 2.5 0 1 0 0-5 2.5 2.5 0 0 0 0 5Z"/><path d="M5 13.375c0 1.036.84 1.875 1.875 1.875h6.25c1.036 0 1.875-.84 1.875-1.875 0-1.036-.937-3.125-5-3.125-4.062 0-5 2.09-5 3.125Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h20v20H0z"/></clipPath></defs></svg>
```

Comment:
I think this is still going to fail our icon checks. 

The fill of `#fff` isn't allowed, only fills of `#5C5F62` are allowed. I think the fill="none" can be removed.

width and height attrs should be removed

I don't think the clip path will pass either. Make sure the icon was exported as a flat icon without extra layers.

Reply:
hey @kyledurand ! 

Trying to make sure I export things correctely [from figma](https://www.figma.com/file/0v0kAm6H8Do13rJj4SpIpp/Email-text-assistant?node-id=1834%3A114214&t=4JVQRIgancMRgNny-1). I removed the background fill, but not sure what I need to do to optimize for the following: 
- width and height attrs should be removed
- I don't think the clip path will pass either. Make sure the icon was exported as a flat icon without extra layers.